src/cdk/v2/destinations/zoho/transformRecord.js

```diff
@@ -351,15 +375,4 @@ const processRecordInputs = async (inputs, destination) => {
   return [...response, ...errorResponseList];
 };
 
-const processRecordInputsWrap = async (inputs, destination) => {
-  if (!inputs || inputs.length === 0) {
-    return [];
-  }
-  const event = inputs[0];
-  if (isEventSentByVDMV2Flow(event)) {
-    return processRecordInputsV2(inputs, destination);
-  }
-  return processRecordInputs(inputs, destination);
-};
-
-module.exports = { processRecordInputsWrap };
+module.exports = { processRecordInputs };
```

Comment:
can we make this refactoring simple ?
let zoho/transformRecordV2.js be there & export `processRecordsInputsV2`. And clean up the code in this file related to processRecordInputs